demos/3d_segmentation_demo/python/3d_segmentation_demo.py

```diff
@@ -262,7 +262,7 @@ def main():
         raise AttributeError("Device {} do not support of 3D convolution. "
                              "Please use CPU, GPU or HETERO:*CPU*, HETERO:*GPU*")
 
-    # --------------------- 2. Read IR Generated by ModelOptimizer (.xml and .bin files) ---------------------
+    # --------------------- 2. Read IR Generated by ModelOptimizer (.xml file) ---------------------
```

Comment:
I suggest remodeling such comments: there is `Reading model` print right below

Reply:
Removed all comments